packages/react-components/react-avatar/src/stories/AvatarGroup/AvatarGroupIndicator.stories.tsx

```diff
@@ -10,32 +15,36 @@ const useStyles = makeStyles({
   },
 });
 
+const names = [
+  'Johnie McConnell',
+  'Allan Munger',
+  'Erik Nason',
+  'Kristin Patterson',
+  'Daisy Phillips',
+  'Carole Poland',
+  'Carlos Slattery',
+  'Robert Tolbert',
+  'Kevin Sturgis',
+  'Charlotte Waltson',
+  'Elliot Woodward',
+];
+
 export const Indicator = () => {
   const styles = useStyles();
+  const countAvatarGroupItems = names.map(name => <AvatarGroupItem name={name} key={'spread-' + name} />);
+  const iconAvatarGroupItems = names.map(name => <AvatarGroupItem name={name} key={'stack-' + name} />);
```

Comment:
(super-nit): `key` only needs to be unique within a given parent element. Since each of the layouts has a different parent, you can just use `key={name}` here:
```suggestion
  const countAvatarGroupItems = names.map(name => <AvatarGroupItem name={name} key={name} />);
  const iconAvatarGroupItems = names.map(name => <AvatarGroupItem name={name} key={name} />);
```

Ditto for the other stories files as well.